featuretools/entityset/relationship.py

```diff
@@ -161,6 +161,13 @@ def __eq__(self, other):
         return isinstance(other, RelationshipPath) and \
             self._relationships_with_direction == other._relationships_with_direction
 
+    def __repr__(self):
+        if self._relationships_with_direction:
+            path = '%s.%s' % (next(self.entities()), self.name)
+        else:
+            path = 'empty'
```

Comment:
to discuss: is this the best string to use?

Reply:
Options:
1. `<RelationshipPath empty>` (current)
2. `<RelationshipPath>`
3. `<RelationshipPath None>`
4. `<RelationshipPath []>`

I'm leaning towards number 4 because `RelationshipPath` generally acts like a list so `[]` makes it clear that this path is like an empty list. Since the normal format is `<RelationshipPath entity1.entity2>`, `<RelationshipPath empty>` could potentially look like `empty` is the name of an entity.

Reply:
i like `[]` too